modules/core/shared/src/main/scala/lucuma/core/model/sequence/Sequence.scala

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model.sequence
+
+import cats.Eq
+import cats.syntax.all._
+import monocle.Focus
+import monocle.Lens
+import monocle.Prism
+import monocle.macros.GenPrism
+
+sealed trait Sequence {
+  val atoms: List[Atom]
+  val time: StepTime
+}
+object Sequence       {
+  final case class GmosNorth(atoms: List[Atom.GmosNorth], time: StepTime) extends Sequence
```

Comment:
`StepTime` will be derived by summing the atom step times.  I think we should keep it separate in the model?  I combined them in the API query results because I assumed the caller would just want to consume it.  Maybe that was wrong.

Reply:
Ok, let's not have redundant information in the model. We could substitute it here for a method that computes it locally.

Reply:
In light of this, I wonder if we should completely remove `Sequence` and replace `Sequence.<Instrument>` with `List[Atom.<Instrument>]` everywhere. WDYT?

Reply:
That's probably fine.